pkl-server/src/main/kotlin/org/pkl/server/Server.kt

```diff
@@ -226,6 +256,9 @@ class Server(private val transport: MessageTransport) : AutoCloseable {
     if (message.clientModuleReaders?.isNotEmpty() == true) {
       add(ClientModuleKeyFactory(message.clientModuleReaders, transport, evaluatorId))
     }
+    for ((scheme, spec) in message.externalModuleReaders ?: emptyMap()) {
+      add(ModuleKeyFactories.external(scheme, getExternalProcess(evaluatorId, spec), evaluatorId))
```

Comment:
Do these external readers ever need to care about what the evaluator ID is?

Reply:
Not really, since the `MessageTransport` is distinct from the server's. One reason to still populate this is `PKL_DEBUG` logging. Because the message contents are printed, this could help debug issues in multi-evaluator scenarios.